test/singer_clojure/messages_test.clj

```diff
@@ -0,0 +1,136 @@
+(ns singer-clojure.messages-test
+  (:require [singer-clojure.messages :as singer-messages]
+            [clojure.test :refer [is deftest]]
+            [clojure.string :as string]
+            [clojure.data.json :as json]))
+
+(defmacro with-out-str-and-value
```

Comment:
Should we add a docstring here to explain the use case / why we added this macro? That we wanted both the stdout AND the returned value.

Reply:
Fixed